src/main/java/org/codeforamerica/shiba/pages/PageController.java

```diff
@@ -221,6 +229,29 @@ ModelAndView getPage(
 
         model.put("county", countyParser.parse(applicationData));
 
+        List<String> zipCode = applicationData.getPagesData().safeGetPageInputValue("homeAddress", "zipCode");
+        if (!zipCode.isEmpty()) {
+            model.put("zipCode", zipCode.get(0));
```

Comment:
Prob unlikely since this is a required field, but `.get(0)`  might return outofbounds exception if the list is empty